### ISSUE TITLE
Test out publishing GitHub docs to LSST the Docs

### ISSUE THREAD

**Issue by heather999** [DarkEnergyScienceCollaboration/Twinkles#251]
LSST The Docs is now available (this would replace using GitHub Pages)
https://community.lsst.org/t/introducing-lsst-the-docs-for-continuous-documentation-delivery/781
Jonathan Sick has offered to help interested parties to get started.  I have contacted him. It appears easy enough to get set up.
One hurdle may be the desire to use Sphinx which prefers ReStructuredText to MarkDown.

**Comment by drphilmarshall** [comment]
Whoops - my last comment on #252 is actually more relevant to this thread. In short, go for it, @heather999 ! :-)

**Comment by drphilmarshall** [comment]
Hey @heather999 - from a quick read of [the docs for LSST the docs](https://ltd-mason.lsst.io/travis.html) it looks like a good start would be to implement a pure sphinx documentation of the Twinkles code. Since Twinkles is a somewhat complex  project, how about we start simple and document `SLTimer` the right way from the start, and then graduate to documenting Twinkles in the same way? I am a sphinx newbie: would you mind helping me get started over at the `SLTimer` repo? I've made an issue in anticipation: https://github.com/DarkEnergyScienceCollaboration/SLTimer/issues/15  Thank you! :-)

**Comment by rbiswas4** [comment]
Or Monitor? We are already putting in docstrings in sphinx format I hope.

**Comment by drphilmarshall** [comment]
Good idea: you should make a corresponding issue for the Monitor right now,
and then Heather can politely decline the SLTimer issue I just assigned her
:-) I am happy to copy the minimal sphinx implemntation over to SLTimer no
problem, I just need an expert to get me started. I'm talking about a
minimal index.rst, Makefile with instructions, one python code file with
correct header and function docstrings, that all compiles into a
reasonable-looking set of HTML files that introduce the code in general and
the API in particular (which I *think* is what we want - right?).

**Comment by drphilmarshall** [comment]
Looking at this with @heather999 , we think we'd like to try setting up Sphinx and running it from travis, having travis push the built html pages back to the gh-pages branch of this repo (which will need setting up). [Here's an example](https://gist.github.com/domenic/ec8b0fc8ab45f39403dd). I'll issue myself with the gh-pages part, @heather999 will set up sphinx :-)
